configs/openmrs_config/ampathforms/NCD_Consultation.json

```diff
@@ -279,7 +279,200 @@
 							"validators": []
 						}
 					]
-				}
+				},
+				{
+					"isExpanded": "true",
+					"label": "Right Renal Exam",
```

Comment:
@kazlaw this should be merged with `Left Renal Exam` to have just one called `Renal Exam`

Reply:
Actually `Renal Tests` and not `Renal Exam`